lib/textstat.rb

```diff
@@ -95,37 +93,35 @@ def self.smog_index(text)
     if sentences >= 3
       begin
         polysyllab = polysyllab_count(text)
-        smog = (
-        (1.043 * (30 * (polysyllab / sentences))**0.5) + 3.1291)
-        return smog.round(1)
+        smog = 1.043 * Math.sqrt(30.0 * polysyllab / sentences) + 3.1291
```

Comment:
`**0.5` and `Math.sqrt` are functionally the same so suggest erring on the side of readability here